CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Adds Storybook configs (#463)
 - Adds vtex search tracking script. With this we will populate TopSearches and Autocomplete indices (#389)
 - Add `RegionalizationBar`, `RegionalizationButton` components and integrates it on Mobile and Desktop devices (#424).
+- New items in the `pull_request_template.md`
```

Comment:
Should we always add the entry at the bottom of the section?

Reply:
My bad, Dan. Thanks for pointing that out! I'm going to fix it to add it at the beginning of the section.